src/types.ts

```diff
@@ -143,6 +143,7 @@ export interface SessionRecordingOptions {
     collectFonts?: boolean
     inlineStylesheet?: boolean
     recorderVersion?: 'v1' | 'v2'
+    recordCrossOriginIframes?: boolean
```

Comment:
Should we warn if someone turns this on but has `v1` as the recorderVersion?

And a bit ouroboros but we should link to the docs and why someone needs to think before turning this on

Reply:
I thought the same but I'm actually going to chat in our standup about rolling out v2 to everyone anyways